### PR TITLE
Corrected the reading logic for ceph version details

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/ceph-integration#112

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Review by mkudlej — APPROVED** [review]
LGTM

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py [hunk 1]
```diff
@@ -83,10 +83,10 @@ def _get_sds_details(self):
             return None
 
         if out:
-            details = json.loads(out)
+            details = out.split()
```
Comment (cybernth) [inline tendrl/ceph_integration/objects/tendrl_context/__init__.py]: Please add comment mentions the output of the ceph --version, which helps th ereview

**Review by cybernth — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/objects/tendrl_context/__init__.py (hunk 1 above)
Comment (shtripat) [reply tendrl/ceph_integration/objects/tendrl_context/__init__.py]: Done. Will add some comments to explain

**Comment by shtripat** [comment]
@Tendrl/tendrl-core plz review

**Review by r0h4n — APPROVED** [review]

**Pull request merged**